web/src/containers/activity/CostAllocateFFP.test.js

```diff
@@ -118,19 +120,12 @@ describe('the CostAllocateFFP component', () => {
       .filterWhere(n => n.props().value === '90-10')
       .simulate('change', { target: { value: '35-65' } });
 
-    expect(
-      props.updateActivity.calledWith(
-        'activity key',
-        { costAllocation: { '1066': { ffp: { federal: 35, state: 65 } } } },
-        true
-      )
-    );
+    expect(props.setFundingSplit).toHaveBeenCalledWith(0, '1066', 35, 65);
   });
 
   test('maps redux state to component props', () => {
     const mapStateToProps = makeMapStateToProps();
-
-    expect(mapStateToProps(state, { aKey: 'key' })).toEqual({
+    expect(mapStateToProps(state, { activityIndex: 0 })).toEqual({
```

Comment:
This is exploding and I can't figure out why. When the test calls `makeSelectCostAllocateFFP` in `activities.selectors.js`, it bombs out on [this line](https://github.com/18F/cms-hitech-apd/blob/feature/document-model/web/src/reducers/activities.selectors.js#L35). Using `node --inspect` in Chrome, it looks like both the `budget` and `activity` are undefined at this point in the test. I've been trying to wrap my head around what might be going wrong with the selector, but I don't have enough brain cells to rub together at 5:30 on a Friday to figure it out, so documenting here.

Reply:
See above for details. This also needs an activity key for the budget part, which is still based on activity keys (whoops!). Rather than try to pull all that out in this PR, you can add the activity key here and (with the fix describe above), that should take care of this:

```js
mapStateToProps(state, { activityIndex: 0, aKey: 'key' })
```